WinRT.Runtime/Projections/IReadOnlyDictionary.net5.cs

```diff
@@ -351,9 +351,31 @@ private ConstantSplittableMap(global::System.Collections.Generic.KeyValuePair<K,
 
                 public uint Size => (uint)(lastItemIndex - firstItemIndex + 1);
 
-                public global::System.Collections.Generic.IEnumerable<K> Keys => throw new NotImplementedException();
+                public global::System.Collections.Generic.IEnumerable<K> Keys  
```

Comment:
I wonder if we should modify DictionaryValueCollection to work also for this scenario and make use of it here similar to in IDictionary.  A couple reasons why I am thinking that is technically Keys and Values collection apparently cannot be modified which DictionaryValueCollection handles, but I think below we will allow that.  And if someone does modify it now, I wonder if we can get into bad states.